CI failure in unit tests. Here's one of the failures:

```
2017-02-21 21:43:35 >           assert generated_facts[key] == val, msg
2017-02-21 21:43:35 E           AssertionError: Comparing value of distribution_release on openSUSE Leap 42.1, should: x86_64, is: 1
2017-02-21 21:43:35 E           assert '1' == 'x86_64'
2017-02-21 21:43:35 E             - 1
2017-02-21 21:43:35 E             + x86_64
2017-02-21 21:43:35 
2017-02-21 21:43:35 test/units/module_utils/test_distribution_version.py:840: AssertionError
```

Full test results here: https://app.shippable.com/runs/58acb34785f0850f0040e0da/2/tests

--- a/test/units/module_utils/test_distribution_version.py
+++ b/test/units/module_utils/test_distribution_version.py
@@ -153,7 +153,7 @@ CODENAME = Malachite
         "result":{
             "distribution": "openSUSE Leap",
             "distribution_major_version": "42",
-            "distribution_release": "x86_64",
+            "distribution_release": "1",
             "os_family": "Suse",
             "distribution_version": "42.1",
         }

--- a/test/units/module_utils/test_distribution_version.py
+++ b/test/units/module_utils/test_distribution_version.py
@@ -179,7 +179,7 @@ ID_LIKE="suse"
         'platform.dist': ('SuSE', '13.2', 'x86_64'),
         'result': {'distribution': u'openSUSE',
                    'distribution_major_version': u'13',
-                   'distribution_release': u'Harlequin',
+                   'distribution_release': u'2',
                    'os_family': u'Suse',
                    'distribution_version': u'13.2'}
     },
@@ -196,7 +196,7 @@ ID_LIKE="suse"
         },
         "name": "openSUSE Tumbleweed 20160917",
         "result": {
-            "distribution_release": "NA",
+            "distribution_release": "",
             "distribution": "openSUSE Tumbleweed",
             "distribution_major_version": "NA",
             "os_family": "Suse",

